vmck/base_settings.py

```diff
@@ -53,7 +55,7 @@
 STATIC_URL = '/static/'
 
 NOMAD_JOB_PREFIX = 'vmck-'
-NOMAD_DEPLOYMENT_NAME = "vmck"
+NOMAD_DEPLOYMENT_NAME = f"vmck@{os.environ.get('HOSTNAME', gethostname())}"
```

Comment:
Extract `hostname` here too. Maybe even move it to `vmck.utils` to avoid duplication.